### PR TITLE
Snippets dir

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
A test of Kramdoc to convert pages from markdown to asciidoc for the purpose of archiving duplicate snippets because we needed the same snippet for both markdown and asciidoc files.

One snippet still exists in both md and adoc as it's included in the config ref and I did not want to convert the config ref at this time.

# Reasons
We needed to test Kramdoc to see if it is a viable solution to convert files to asciidoc.
- Links require additional formatting
- Code snippets require additional formatting
- Snippets require additional formatting
- Tables are not converted
- Front matter needs formatting
- No file I tested included an image, so that remains unchecked

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Comment by devinmarieb** [comment]
This is currently failing on `Could not locate the included file 'snippets/pipeline-values.adoc'` - however I have double checked and that file does exist in the right location and there is no typo. Not sure why it can't find this file.

File: jekyll/archived/snippets/built-in-env-vars.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Variable | Type | Value
```
Comment (rosieyohannan) [inline jekyll/archived/snippets/built-in-env-vars.md:1]: I think as well as being added to the archived folder this one still needs deleting from `_includes/snippets`?

File: jekyll/archived/snippets/language-guides-two-copy.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+| Language Guide                          | Framework    | GitHub Repo Name
```
Comment (rosieyohannan) [inline jekyll/archived/snippets/language-guides-two-copy.md:1]: I think as well as being added to the archived folder this one still needs deleting from `_includes/snippets`?

File: jekyll/_cci2/android-machine-image.adoc [hunk 3]
```diff
@@ -1,51 +1,44 @@
 ---
```
Comment (rosieyohannan) [inline jekyll/_cci2/android-machine-image.adoc:1]: just an observation: it seems odd that this file is showing as an edited file, whereas the other conversions are shown as entirely new files 🤔

**Comment by devinmarieb** [comment]
@rosieyohannan Huh, I've never had to additionally delete a file after manually moving it from the cci2 directory to the archive folder before, it's always moved location. I just tried again on a test file and it removed itself from the directory when I moved it to archive. It seems it somehow magically replaces itself though. I have no idea how or why that's happening. 

I'm not sure I know what you mean about the android page. When I view the commits they all show `.md -> .adoc`

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**